README.md

```diff
@@ -64,7 +64,7 @@ python ./combine.py ./some/path/to/allure/generated/folder --remove-temp-files
 ## Import and use in python code
 
 ```python
-from combine import combine_allure
+from allure2html.combine import combine_allure
```

Comment:
By the way, how would the module be named in imports? We could use allure_combine or allurecombine (not allure-combine, unfortunatelly)

Reply:
Yes, it will raise an error, when use allure-combine.

I will use allure_combine, is that OK?

Reply:
> Yes, it will raise an error, when use allure-combine.
> 
> I will use allure_combine, is that OK?

or any other names?

Reply:
Ok